xapian-core/net/remote_protocol.rst

```diff
@@ -318,3 +318,33 @@ Reconstruct text
 
 -  ``MSG_RECONSTRUCTTEXT I<did> I<length> I<start_pos> I<end_pos> <prefix>``
 -  ``REPLY_RECONSTRUCTTEXT <text>``
+
+Synonym Term List
+-----------------
+
+-  ``MSG_SYNONYMTERMLIST <word>``
+-  ``REPLY_SYNONYMTERMLIST [C<chars of previous term to reuse> S<string to append>]...``
```

Comment:
This doesn't match what's actually implemented (the count is missing).  Same below.